src/pages/Scholarships.jsx

```diff
@@ -148,9 +149,9 @@ function ScholarshipsPage() {
             <Button
               className={classes.loadMoreButton}
               color="primary"
-              disabled={disableMoreBtn}
+              disabled={canLoadMore}
               onClick={() => loadMoreScholarships(loadMoreFn)}>
-              Load More
+              {canLoadMore ? 'No more results' : 'Load More'}
```

Comment:
This still shows the button. I'd rather it be something like:
```jsx
{canLoadMore ? <Button>... : 'No more results'}
```

Here and in the other page.